src/tests/Microsoft.PowerFx.Interpreter.Tests/DisplayNameTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class DisplayNameTests
+    {
+        [Fact]
+        public void DisplayNameTest1()
```

Comment:
please add tests covering Engine.GetInvariantExpression/GetDisplayExpression

See the DisplayNameTests in Core for examples